src/main/java/com/machowina/model/User.java

```diff
@@ -0,0 +1,46 @@
+package com.machowina.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+import org.hibernate.validator.constraints.NotBlank;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Data
+@NoArgsConstructor
+@Entity
+public class User {
+	
+	@Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+	
+	@NotBlank
+    private String username;
+	
+	@NotBlank
+	private String password;
```

Comment:
czy hasło jest zabezpieczone?

Reply:
Powinno być zahashowane np. BCryptem, ale nie robiłam tu w ogóle dodawania userów więc nie jest to nigdzie zaimplementowane